linux/library/src/flutter_embedder.cc

```diff
@@ -81,13 +81,16 @@ static void GLFWOnFlutterPlatformMessage(const FlutterPlatformMessage *message,
     return;
   }
   GLFWwindow *window = reinterpret_cast<GLFWwindow *>(user_data);
-  Json::Reader parser;
+  Json::CharReaderBuilder reader_builder;
+  std::unique_ptr<Json::CharReader> const parser(reader_builder.newCharReader());
```

Comment:
Remove the `const` here:

```c++
std::unique_ptr<Json::CharReader> parser(reader_builder.newCharReader());
``` 

Should be fine.

Reply:
Fixed